src/index.js

```diff
@@ -0,0 +1,24 @@
+import { getRange, getPeriodParams, getCompareRange, getAutoCompareRangeAndLabel, getTillYesterdayPeriod, getCustomPeriod, getLastPeriod, getThisPeriod } from './main';
+import { migrateLegacyPeriod, migrateLegacyCompareMode, toLegacyPeriod, toLegacyCompareMode, retrievePeriod, retrievePeriodParams, retrieveComparePeriod, calculateAutoCompare } from './legacy';
+import { formatDate } from './utils';
+
+export default {
+  getRange,
+  getPeriodParams,
+  getCompareRange,
+  getAutoCompareRangeAndLabel,
+  getTillYesterdayPeriod,
+  getCustomPeriod,
+  getLastPeriod,
+  getThisPeriod,
+  formatDate,
+  // legacy
+  migrateLegacyPeriod,
+  migrateLegacyCompareMode,
+  toLegacyPeriod,
+  toLegacyCompareMode,
+  retrievePeriod,
+  retrievePeriodParams,
+  retrieveComparePeriod,
+  calculateAutoCompare,
```

Comment:
So all methods names have changed?

Reply:
Could you write extensive documentation about the changes you're introducing to the API?
Other services also depend on this library (off the top of my head, the app and innovation)

Reply:
I'm writing the changelog